components/admin_console/schema_admin_settings.jsx

```diff
@@ -349,7 +349,7 @@ export default class SchemaAdminSettings extends React.Component {
         return (
             <RequestButton
                 key={this.props.schema.id + '_text_' + setting.key}
-                requestAction={setting.action}
+                requestAction={(success, error) => setting.action(success, error, this.state)}
```

Comment:
ahh didn't realize this is how it was being set up. I think this should remain `requestAction={setting.action}`

Reply:
I moved this up to its own function expression.